tests/characterization_data/filters_expected/sample_nsxv.nsx

```diff
@@ -1,54 +1,612 @@
-<!--
- $Id:$
- $Date:$
- $Revision:$
--->
+<?xml version="1.0" ?>
```

Comment:
You can remove this I believe. You would need to serialize the xml to a string using the [toxml function](https://docs.python.org/2/library/xml.dom.minidom.html#xml.dom.minidom.Node.toxml)

@SecurityForDotes is this header a problem or is it just a nit?